src/mds/Locker.cc

```diff
@@ -2991,6 +2992,10 @@ void Locker::_update_cap_fields(CInode *in, int dirty, MClientCaps *m, inode_t *
 	      << " for " << *in << dendl;
       pi->mtime = mtime;
     }
+    if (((dirty & CEPH_CAP_FILE_WR) && change_attr > pi->change_attr) ||
+	((dirty & CEPH_CAP_FILE_EXCL) && change_attr != pi->change_attr)) {
+      pi->change_attr = change_attr;
```

Comment:
At a quick thought, it seems like it needs to be ceiling+1 if multiple clients bumped it. Because a NFS client of a Ganesha instance that is thusly a ceph client that happened to have the ceiling change attr needs to see it change again to realize the other ceph clients ALSO changed the file...